react/features/notifications/actions.js

```diff
@@ -14,11 +14,34 @@ import {
     SHOW_NOTIFICATION
 } from './actionTypes';
 import {
+    NOTIFICATION_TIMEOUT_TYPE,
     NOTIFICATION_TIMEOUT,
     NOTIFICATION_TYPE,
     SILENT_JOIN_THRESHOLD
 } from './constants';
 
+declare var APP: Object;
+
+/**
+ * Function that returns notification timeout value based on notification timeout type.
+ *
+ * @param {string} type - Notification type.
+ * @returns {number}
+ */
+function getNotificationTimeout(type: ?string) {
+    const { notificationTimeouts } = APP.store.getState()['features/base/config'];
```

Comment:
This is not going to work on mobile, you can't access APP there. You should get a reference to the state as a  parameter and extract the timeouts from there.